examples/custom-widget-with-1st-party-data/src/controller/custom-widget.ts

```diff
@@ -0,0 +1,7 @@
+import { DDClient } from '@datadog/ui-extensions-sdk';
+
+/**
+ * Here we do any initialization of we need for the custom widget.
+ * This could include listening to events from the main controller
+ */
+export function setupCustomWidget(client: DDClient): void {}
```

Comment:
Do you have specific ideas what is needed here? We could remove it until it's needed

I have one idea but we can't do it until we complete the widget options work you started earlier this querter: Dyamic widget options

Reply:
Yeah, that was the main thing I was thinking. But yeah, probably not worth hanging onto this.

Would the `api_access_change` event be worthwhile to illustrate here? It is about 1st-party auth (as far as I understand).

Reply:
Whether we keep this or not, I realize it might actually be better to have these setup functions as `useSetup*` hooks that actually do the right thing as far as unsubscribing of listeners.

Reply:
Eh, let's just dump it. The other templates can show this pattern, and if we ever move on UIAPPS-90, we can follow that pattern.